5.0.0-rc1 terraform provider  has error

### Confirmation

- [x] This is a bug with an existing resource and is not a feature request or enhancement. Feature requests should be submitted with Cloudflare Support or your account team.
- [x] I have searched the issue tracker and my issue isn't already found.
- [x] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

When i use 5.0.0-rc1 terraform provider , it returns errors like below.
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: startup_time_ms, etag, has_assets, placement_mode, modified_on, has_modules, and placement_status.



### Affected resource(s)

resource cloudflare_workers_script

### Terraform configuration files

```hcl
resource "cloudflare_workers_script" "example_workers_script" {
  account_id = "xxxxxxxx"
  script_name = "test"
    content    = file("index.js")

    assets = {
      config = {
        html_handling = "auto-trailing-slash"
        not_found_handling = "none"
        serve_directly = true
      }
      jwt = "jwt"
    }
    bindings = [{
      name = "MY_ENV_VAR"
      type = "plain_text"
    }]
    compatibility_date = "2025-01-10"
    compatibility_flags = ["nodejs_compat_v2"]


 
  
}
```

### Link to debug output

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

2025-01-15T16:31:15.018+0800 [INFO]  backend/local: apply calling Apply
2025-01-15T16:31:15.018+0800 [DEBUG] Building and walking apply graph for NormalMode plan
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_kv_namespace.workers_kv_namespace", instance cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_script.example_workers_script", instance cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "output.name (expand)" references: [cloudflare_workers_kv_namespace.workers_kv_namespace (expand) cloudflare_workers_kv_namespace.workers_kv_namespace cloudflare_workers_kv_namespace.workers_kv_namespace]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" references: []
2025-01-15T16:31:15.020+0800 [DEBUG] Starting graph walk: walkApply
2025-01-15T16:31:15.021+0800 [DEBUG] created provider logger: level=debug
2025-01-15T16:31:15.021+0800 [INFO]  provider: configuring client automatic mTLS
2025-01-15T16:31:15.025+0800 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1]
2025-01-15T16:31:15.030+0800 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:15.030+0800 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1
2025-01-15T16:31:15.053+0800 [INFO]  provider.terraform-provider-cloudflare_v5.0.0-rc1: configuring server automatic mTLS: timestamp=2025-01-15T16:31:15.053+0800
2025-01-15T16:31:15.060+0800 [DEBUG] provider: using plugin: version=6
2025-01-15T16:31:15.060+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: plugin address: address=/var/folders/k6/n67v29s97t9g64gp2zd081m80000gq/T/plugin585257130 network=unix timestamp=2025-01-15T16:31:15.060+0800
2025-01-15T16:31:15.107+0800 [DEBUG] No provider meta schema returned
2025-01-15T16:31:15.122+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_kv_namespace @module=sdk.framework tf_attribute_path=AttributeName("id") tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("supports_url_encoding") tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creating...
2025-01-15T16:31:15.125+0800 [INFO]  Starting apply for cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.126+0800 [DEBUG] cloudflare_workers_kv_namespace.workers_kv_namespace: applying the planned Create change
2025-01-15T16:31:15.127+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
POST /client/v4/accounts/xxxxxxxx/storage/kv/namespaces HTTP/1.1
> x-stainless-lang: Terraform
> accept: application/json
> x-stainless-package-version: 5.0.0-rc1
> x-stainless-runtime-version: 1.13.0
> x-stainless-retry-count: 0
> user-agent: terraform-provider-cloudflare/5.0.0-rc1 terraform-plugin-framework/1.13.0 terraform/1.5.7
> x-stainless-runtime: terraform-plugin-framework
> x-stainless-os: MacOS
> authorization: [redacted]
> content-type: application/json
> x-stainless-arch: arm64
>

{"title":"test"}

: tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:64 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:15.127+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 @module=sdk.framework tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("etag") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("placement") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("placement_status") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("usage_model") @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("tail_consumers") tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("placement_mode") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("modified_on") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_attribute_path=AttributeName("startup_time_ms") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("has_modules") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework tf_attribute_path=AttributeName("id") @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_attribute_path=AttributeName("observability") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("has_assets") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 @module=sdk.framework tf_attribute_path=AttributeName("migrations") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_attribute_path=AttributeName("logpush") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
cloudflare_workers_script.example_workers_script: Creating...
2025-01-15T16:31:15.142+0800 [INFO]  Starting apply for cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.142+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.143+0800 [DEBUG] cloudflare_workers_script.example_workers_script: applying the planned Create change
2025-01-15T16:31:15.145+0800 [ERROR] provider.terraform-provider-cloudflare_v5.0.0-rc1: Response contains error diagnostic: @caller=github.com/hashicorp/terraform-plugin-go@v0.25.0/tfprotov6/internal/diag/diagnostics.go:58 diagnostic_attribute= tf_proto_version=6.7 tf_rpc=ApplyResourceChange @module=sdk.proto tf_provider_addr=registry.terraform.io/cloudflare/cloudflare diagnostic_detail="An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the provider developer:

mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets." diagnostic_severity=ERROR diagnostic_summary="Value Conversion Error" tf_req_id=2020e7ea-d4e6-d2ce-f8fc-a1470d666433 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.145+0800
2025-01-15T16:31:15.155+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
2025-01-15T16:31:15.155+0800 [ERROR] vertex "cloudflare_workers_script.example_workers_script" error: Value Conversion Error
2025-01-15T16:31:19.366+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
< HTTP/2.0 200 OK
< date: Wed, 15 Jan 2025 08:31:19 GMT
< cf-cache-status: DYNAMIC
< set-cookie: __cflb=0H28vgHxwvgAQtjUGU4yFBDJQfw1pfzuUqbBSnb5jw3; SameSite=Lax; path=/; expires=Wed, 15-Jan-25 11:01:20 GMT; HttpOnly
< set-cookie: __cf_bm=6I6CLtjY.NLg3ZWAEUTK7bab5gl4jOi_l5VksZSSj8k-1736929879-1.0.1.1-NUMUtSaEWPi55nJrepEg7K7Y4sgDjrg_VhXSnluta9wK8LHIvMgQYhc.0pfpzB2YIRlr9jfZ.p25f0ehZsk7FA; path=/; expires=Wed, 15-Jan-25 09:01:19 GMT; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< set-cookie: __cfruid=7ee4140f208f57e8fa84a86497998e7e62bce786-1736929879; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< vary: Accept-Encoding
< server: cloudflare
< content-type: application/json; charset=UTF-8
< cf-ray: 9024822fcb409cd1-SIN
< cache-control: no-store, no-cache, must-revalidate
< cf-auditlog-id: 01946916-99f6-7e72-9e6a-226b0df3cc37
<

{
  "result": {
    "id": "c7d588be7e354bb88f223b3db2dd8f25",
    "title": "test",
    "supports_url_encoding": true
  },
  "success": true,
  "errors": [],
  "messages": []
}


: tf_rpc=ApplyResourceChange @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:92 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:19.366+0800
2025-01-15T16:31:19.367+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 tf_attribute_path=title tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:19.366+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creation complete after 4s [id=c7d588be7e354bb88f223b3db2dd8f25]
2025-01-15T16:31:19.370+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_attribute_path=account_id @module=sdk.framework tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 timestamp=2025-01-15T16:31:19.367+0800
2025-01-15T16:31:19.376+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
╷
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets.
╵
2025-01-15T16:31:19.389+0800 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin exited

### Panic output

cloudflare_workers_kv_namespace.workers_kv_namespace: Creating...
cloudflare_workers_script.example_workers_script: Creating...
cloudflare_workers_kv_namespace.workers_kv_namespace: Creation complete after 5s [id=969df8d80dbb4587902740aedc58e1ef]
╷
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: startup_time_ms, etag, has_assets, placement_mode, modified_on, has_modules, and placement_status.
╵

### Expected output

terraform apply successfully

### Actual output

cloudflare_workers_kv_namespace.workers_kv_namespace: Creating...
cloudflare_workers_script.example_workers_script: Creating...
cloudflare_workers_kv_namespace.workers_kv_namespace: Creation complete after 5s [id=969df8d80dbb4587902740aedc58e1ef]
╷
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: startup_time_ms, etag, has_assets, placement_mode, modified_on, has_modules, and placement_status.
╵

### Steps to reproduce

1.  add provider.tf 
`terraform {
  required_providers {
    cloudflare = {
      source  = "cloudflare/cloudflare"
      version = "5.0.0-rc1"
    }
  }
}

provider "cloudflare" {
  api_token = "xxxxxx"
  
}`
2. add main.tf
`
resource "cloudflare_workers_kv_namespace" "workers_kv_namespace" {
  account_id = "xxxxxxxx"
  title = "test"
  
}



resource "cloudflare_workers_script" "example_workers_script" {
  account_id = "xxxxxxxx"
  script_name = "test"
    content    = file("index.js")

    assets = {
      config = {
        html_handling = "auto-trailing-slash"
        not_found_handling = "none"
        serve_directly = true
      }
      jwt = "jwt"
    }
    bindings = [{
      name = "MY_ENV_VAR"
      type = "plain_text"
    }]
    compatibility_date = "2025-01-10"
    compatibility_flags = ["nodejs_compat_v2"]


 
  
}`

3. terrraform  init && terraform apply

### Additional factoids

_No response_

### References

_No response_

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

2025-01-15T16:31:15.018+0800 [INFO]  backend/local: apply calling Apply
2025-01-15T16:31:15.018+0800 [DEBUG] Building and walking apply graph for NormalMode plan
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_kv_namespace.workers_kv_namespace", instance cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_script.example_workers_script", instance cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "output.name (expand)" references: [cloudflare_workers_kv_namespace.workers_kv_namespace (expand) cloudflare_workers_kv_namespace.workers_kv_namespace cloudflare_workers_kv_namespace.workers_kv_namespace]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" references: []
2025-01-15T16:31:15.020+0800 [DEBUG] Starting graph walk: walkApply
2025-01-15T16:31:15.021+0800 [DEBUG] created provider logger: level=debug
2025-01-15T16:31:15.021+0800 [INFO]  provider: configuring client automatic mTLS
2025-01-15T16:31:15.025+0800 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1]
2025-01-15T16:31:15.030+0800 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:15.030+0800 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1
2025-01-15T16:31:15.053+0800 [INFO]  provider.terraform-provider-cloudflare_v5.0.0-rc1: configuring server automatic mTLS: timestamp=2025-01-15T16:31:15.053+0800
2025-01-15T16:31:15.060+0800 [DEBUG] provider: using plugin: version=6
2025-01-15T16:31:15.060+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: plugin address: address=/var/folders/k6/n67v29s97t9g64gp2zd081m80000gq/T/plugin585257130 network=unix timestamp=2025-01-15T16:31:15.060+0800
2025-01-15T16:31:15.107+0800 [DEBUG] No provider meta schema returned
2025-01-15T16:31:15.122+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_kv_namespace @module=sdk.framework tf_attribute_path=AttributeName("id") tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("supports_url_encoding") tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creating...
2025-01-15T16:31:15.125+0800 [INFO]  Starting apply for cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.126+0800 [DEBUG] cloudflare_workers_kv_namespace.workers_kv_namespace: applying the planned Create change
2025-01-15T16:31:15.127+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
POST /client/v4/accounts/xxxxxxxx/storage/kv/namespaces HTTP/1.1
> x-stainless-lang: Terraform
> accept: application/json
> x-stainless-package-version: 5.0.0-rc1
> x-stainless-runtime-version: 1.13.0
> x-stainless-retry-count: 0
> user-agent: terraform-provider-cloudflare/5.0.0-rc1 terraform-plugin-framework/1.13.0 terraform/1.5.7
> x-stainless-runtime: terraform-plugin-framework
> x-stainless-os: MacOS
> authorization: [redacted]
> content-type: application/json
> x-stainless-arch: arm64
>

{"title":"website-category-test"}

: tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:64 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:15.127+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 @module=sdk.framework tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("etag") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("placement") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("placement_status") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("usage_model") @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("tail_consumers") tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("placement_mode") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("modified_on") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_attribute_path=AttributeName("startup_time_ms") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("has_modules") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework tf_attribute_path=AttributeName("id") @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_attribute_path=AttributeName("observability") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("has_assets") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 @module=sdk.framework tf_attribute_path=AttributeName("migrations") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_attribute_path=AttributeName("logpush") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
cloudflare_workers_script.example_workers_script: Creating...
2025-01-15T16:31:15.142+0800 [INFO]  Starting apply for cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.142+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.143+0800 [DEBUG] cloudflare_workers_script.example_workers_script: applying the planned Create change
2025-01-15T16:31:15.145+0800 [ERROR] provider.terraform-provider-cloudflare_v5.0.0-rc1: Response contains error diagnostic: @caller=github.com/hashicorp/terraform-plugin-go@v0.25.0/tfprotov6/internal/diag/diagnostics.go:58 diagnostic_attribute= tf_proto_version=6.7 tf_rpc=ApplyResourceChange @module=sdk.proto tf_provider_addr=registry.terraform.io/cloudflare/cloudflare diagnostic_detail="An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the provider developer:

mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets." diagnostic_severity=ERROR diagnostic_summary="Value Conversion Error" tf_req_id=2020e7ea-d4e6-d2ce-f8fc-a1470d666433 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.145+0800
2025-01-15T16:31:15.155+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
2025-01-15T16:31:15.155+0800 [ERROR] vertex "cloudflare_workers_script.example_workers_script" error: Value Conversion Error
2025-01-15T16:31:19.366+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
< HTTP/2.0 200 OK
< date: Wed, 15 Jan 2025 08:31:19 GMT
< cf-cache-status: DYNAMIC
< set-cookie: __cflb=0H28vgHxwvgAQtjUGU4yFBDJQfw1pfzuUqbBSnb5jw3; SameSite=Lax; path=/; expires=Wed, 15-Jan-25 11:01:20 GMT; HttpOnly
< set-cookie: __cf_bm=6I6CLtjY.NLg3ZWAEUTK7bab5gl4jOi_l5VksZSSj8k-1736929879-1.0.1.1-NUMUtSaEWPi55nJrepEg7K7Y4sgDjrg_VhXSnluta9wK8LHIvMgQYhc.0pfpzB2YIRlr9jfZ.p25f0ehZsk7FA; path=/; expires=Wed, 15-Jan-25 09:01:19 GMT; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< set-cookie: __cfruid=7ee4140f208f57e8fa84a86497998e7e62bce786-1736929879; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< vary: Accept-Encoding
< server: cloudflare
< content-type: application/json; charset=UTF-8
< cf-ray: 9024822fcb409cd1-SIN
< cache-control: no-store, no-cache, must-revalidate
< cf-auditlog-id: 01946916-99f6-7e72-9e6a-226b0df3cc37
<

{
  "result": {
    "id": "c7d588be7e354bb88f223b3db2dd8f25",
    "title": "website-category-test",
    "supports_url_encoding": true
  },
  "success": true,
  "errors": [],
  "messages": []
}


: tf_rpc=ApplyResourceChange @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:92 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:19.366+0800
2025-01-15T16:31:19.367+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 tf_attribute_path=title tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:19.366+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creation complete after 4s [id=c7d588be7e354bb88f223b3db2dd8f25]
2025-01-15T16:31:19.370+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_attribute_path=account_id @module=sdk.framework tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 timestamp=2025-01-15T16:31:19.367+0800
2025-01-15T16:31:19.376+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
╷
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets.
╵
2025-01-15T16:31:19.389+0800 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin exited

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

2025-01-15T16:31:15.018+0800 [INFO]  backend/local: apply calling Apply
2025-01-15T16:31:15.018+0800 [DEBUG] Building and walking apply graph for NormalMode plan
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_kv_namespace.workers_kv_namespace", instance cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.018+0800 [DEBUG] Resource state not found for node "cloudflare_workers_script.example_workers_script", instance cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.018+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_script.example_workers_script (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ProviderTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/cloudflare/cloudflare"]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script (expand)" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "output.name (expand)" references: [cloudflare_workers_kv_namespace.workers_kv_namespace (expand) cloudflare_workers_kv_namespace.workers_kv_namespace cloudflare_workers_kv_namespace.workers_kv_namespace]
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_kv_namespace.workers_kv_namespace" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "cloudflare_workers_script.example_workers_script" references: []
2025-01-15T16:31:15.019+0800 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/cloudflare/cloudflare\"]" references: []
2025-01-15T16:31:15.020+0800 [DEBUG] Starting graph walk: walkApply
2025-01-15T16:31:15.021+0800 [DEBUG] created provider logger: level=debug
2025-01-15T16:31:15.021+0800 [INFO]  provider: configuring client automatic mTLS
2025-01-15T16:31:15.025+0800 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 args=[.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1]
2025-01-15T16:31:15.030+0800 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:15.030+0800 [DEBUG] provider: waiting for RPC address: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1
2025-01-15T16:31:15.053+0800 [INFO]  provider.terraform-provider-cloudflare_v5.0.0-rc1: configuring server automatic mTLS: timestamp=2025-01-15T16:31:15.053+0800
2025-01-15T16:31:15.060+0800 [DEBUG] provider: using plugin: version=6
2025-01-15T16:31:15.060+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: plugin address: address=/var/folders/k6/n67v29s97t9g64gp2zd081m80000gq/T/plugin585257130 network=unix timestamp=2025-01-15T16:31:15.060+0800
2025-01-15T16:31:15.107+0800 [DEBUG] No provider meta schema returned
2025-01-15T16:31:15.122+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_kv_namespace @module=sdk.framework tf_attribute_path=AttributeName("id") tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
2025-01-15T16:31:15.125+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("supports_url_encoding") tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=887d290a-a7bc-4bb0-c100-debb56a704c6 timestamp=2025-01-15T16:31:15.125+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creating...
2025-01-15T16:31:15.125+0800 [INFO]  Starting apply for cloudflare_workers_kv_namespace.workers_kv_namespace
2025-01-15T16:31:15.126+0800 [DEBUG] cloudflare_workers_kv_namespace.workers_kv_namespace: applying the planned Create change
2025-01-15T16:31:15.127+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
POST /client/v4/accounts/xxxxxxxx/storage/kv/namespaces HTTP/1.1
> x-stainless-lang: Terraform
> accept: application/json
> x-stainless-package-version: 5.0.0-rc1
> x-stainless-runtime-version: 1.13.0
> x-stainless-retry-count: 0
> user-agent: terraform-provider-cloudflare/5.0.0-rc1 terraform-plugin-framework/1.13.0 terraform/1.5.7
> x-stainless-runtime: terraform-plugin-framework
> x-stainless-os: MacOS
> authorization: [redacted]
> content-type: application/json
> x-stainless-arch: arm64
>

{"title":"test"}

: tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:64 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:15.127+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Marking Computed attributes with null configuration values as unknown (known after apply) in the plan to prevent potential Terraform errors: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:217 @module=sdk.framework tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("etag") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("placement") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("placement_status") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("usage_model") @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.135+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("tail_consumers") tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_attribute_path=AttributeName("placement_mode") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("modified_on") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_attribute_path=AttributeName("startup_time_ms") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.135+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_attribute_path=AttributeName("has_modules") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @module=sdk.framework tf_attribute_path=AttributeName("id") @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @module=sdk.framework tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_attribute_path=AttributeName("observability") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 @module=sdk.framework tf_attribute_path=AttributeName("has_assets") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 @module=sdk.framework tf_attribute_path=AttributeName("migrations") tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_resource_type=cloudflare_workers_script tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
2025-01-15T16:31:15.136+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: marking computed attribute that is null in the config as unknown: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwserver/server_planresourcechange.go:467 tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_resource_type=cloudflare_workers_script @module=sdk.framework tf_attribute_path=AttributeName("logpush") tf_req_id=ca7ef130-d6fb-93d7-22d4-b49d834a0132 tf_rpc=PlanResourceChange timestamp=2025-01-15T16:31:15.136+0800
cloudflare_workers_script.example_workers_script: Creating...
2025-01-15T16:31:15.142+0800 [INFO]  Starting apply for cloudflare_workers_script.example_workers_script
2025-01-15T16:31:15.142+0800 [DEBUG] skipping FixUpBlockAttrs
2025-01-15T16:31:15.143+0800 [DEBUG] cloudflare_workers_script.example_workers_script: applying the planned Create change
2025-01-15T16:31:15.145+0800 [ERROR] provider.terraform-provider-cloudflare_v5.0.0-rc1: Response contains error diagnostic: @caller=github.com/hashicorp/terraform-plugin-go@v0.25.0/tfprotov6/internal/diag/diagnostics.go:58 diagnostic_attribute= tf_proto_version=6.7 tf_rpc=ApplyResourceChange @module=sdk.proto tf_provider_addr=registry.terraform.io/cloudflare/cloudflare diagnostic_detail="An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the provider developer:

mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets." diagnostic_severity=ERROR diagnostic_summary="Value Conversion Error" tf_req_id=2020e7ea-d4e6-d2ce-f8fc-a1470d666433 tf_resource_type=cloudflare_workers_script timestamp=2025-01-15T16:31:15.145+0800
2025-01-15T16:31:15.155+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
2025-01-15T16:31:15.155+0800 [ERROR] vertex "cloudflare_workers_script.example_workers_script" error: Value Conversion Error
2025-01-15T16:31:19.366+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: 
< HTTP/2.0 200 OK
< date: Wed, 15 Jan 2025 08:31:19 GMT
< cf-cache-status: DYNAMIC
< set-cookie: __cflb=0H28vgHxwvgAQtjUGU4yFBDJQfw1pfzuUqbBSnb5jw3; SameSite=Lax; path=/; expires=Wed, 15-Jan-25 11:01:20 GMT; HttpOnly
< set-cookie: __cf_bm=6I6CLtjY.NLg3ZWAEUTK7bab5gl4jOi_l5VksZSSj8k-1736929879-1.0.1.1-NUMUtSaEWPi55nJrepEg7K7Y4sgDjrg_VhXSnluta9wK8LHIvMgQYhc.0pfpzB2YIRlr9jfZ.p25f0ehZsk7FA; path=/; expires=Wed, 15-Jan-25 09:01:19 GMT; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< set-cookie: __cfruid=7ee4140f208f57e8fa84a86497998e7e62bce786-1736929879; path=/; domain=.api.cloudflare.com; HttpOnly; Secure; SameSite=None
< vary: Accept-Encoding
< server: cloudflare
< content-type: application/json; charset=UTF-8
< cf-ray: 9024822fcb409cd1-SIN
< cache-control: no-store, no-cache, must-revalidate
< cf-auditlog-id: 01946916-99f6-7e72-9e6a-226b0df3cc37
<

{
  "result": {
    "id": "c7d588be7e354bb88f223b3db2dd8f25",
    "title": "test",
    "supports_url_encoding": true
  },
  "success": true,
  "errors": [],
  "messages": []
}


: tf_rpc=ApplyResourceChange @caller=github.com/cloudflare/terraform-provider-cloudflare/internal/logging/logging.go:92 @module=cloudflare tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace timestamp=2025-01-15T16:31:19.366+0800
2025-01-15T16:31:19.367+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 tf_attribute_path=title tf_provider_addr=registry.terraform.io/cloudflare/cloudflare @module=sdk.framework tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_rpc=ApplyResourceChange timestamp=2025-01-15T16:31:19.366+0800
cloudflare_workers_kv_namespace.workers_kv_namespace: Creation complete after 4s [id=c7d588be7e354bb88f223b3db2dd8f25]
2025-01-15T16:31:19.370+0800 [DEBUG] provider.terraform-provider-cloudflare_v5.0.0-rc1: Value switched to prior value due to semantic equality logic: tf_provider_addr=registry.terraform.io/cloudflare/cloudflare tf_req_id=9f6db453-4b8b-a002-1cf6-4bda46761289 tf_resource_type=cloudflare_workers_kv_namespace tf_attribute_path=account_id @module=sdk.framework tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-framework@v1.13.0/internal/fwschemadata/value_semantic_equality.go:91 timestamp=2025-01-15T16:31:19.367+0800
2025-01-15T16:31:19.376+0800 [DEBUG] State storage *statemgr.Filesystem declined to persist a state snapshot
╷
│ Error: Value Conversion Error
│ 
│   with cloudflare_workers_script.example_workers_script,
│ An unexpected error was encountered trying to convert tftypes.Value into workers_script.WorkersScriptModel. This is always an error in the provider. Please report the following to the
│ provider developer:
│ 
│ mismatch between struct and object: Object defines fields not found in struct: etag, placement_status, modified_on, startup_time_ms, placement_mode, has_modules, and has_assets.
╵
2025-01-15T16:31:19.389+0800 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/cloudflare/cloudflare/5.0.0-rc1/darwin_arm64/terraform-provider-cloudflare_v5.0.0-rc1 pid=43261
2025-01-15T16:31:19.392+0800 [DEBUG] provider: plugin exited

--- a/internal/services/workers_script/model.go
+++ b/internal/services/workers_script/model.go
@@ -8,6 +8,7 @@ import (
 
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/apijson"
 	"github.com/cloudflare/terraform-provider-cloudflare/internal/customfield"
+	"github.com/hashicorp/terraform-plugin-framework-timetypes/timetypes"
 	"github.com/hashicorp/terraform-plugin-framework/types"
 	"github.com/jinzhu/copier"
 )
@@ -17,17 +18,16 @@ type WorkersScriptResultEnvelope struct {
 }
 
 type WorkersScriptModel struct {
-	ID         types.String `tfsdk:"id" json:"-,computed"`
-	ScriptName types.String `tfsdk:"script_name" path:"script_name,required"`
-	AccountID  types.String `tfsdk:"account_id" path:"account_id,required"`
-	// Message    types.String `tfsdk:"message" json:"message,optional"`
-	Content types.String `tfsdk:"content" json:"content,required"`
-	// CreatedOn     timetypes.RFC3339 `tfsdk:"created_on" json:"created_on,computed" format:"date-time"`
-	// Etag          types.String      `tfsdk:"etag" json:"etag,computed"`
-	// HasAssets     types.Bool        `tfsdk:"has_assets" json:"has_assets,computed"`
-	// HasModules    types.Bool        `tfsdk:"has_modules" json:"has_modules,computed"`
-	// ModifiedOn    timetypes.RFC3339 `tfsdk:"modified_on" json:"modified_on,computed" format:"date-time"`
-	// StartupTimeMs types.Int64       `tfsdk:"startup_time_ms" json:"startup_time_ms,computed"`
+	ID            types.String      `tfsdk:"id" json:"-,computed"`
+	ScriptName    types.String      `tfsdk:"script_name" path:"script_name,required"`
+	AccountID     types.String      `tfsdk:"account_id" path:"account_id,required"`
+	Content       types.String      `tfsdk:"content" json:"content,required"`
+	CreatedOn     timetypes.RFC3339 `tfsdk:"created_on" json:"created_on,computed" format:"date-time"`
+	Etag          types.String      `tfsdk:"etag" json:"etag,computed"`
+	HasAssets     types.Bool        `tfsdk:"has_assets" json:"has_assets,computed"`
+	HasModules    types.Bool        `tfsdk:"has_modules" json:"has_modules,computed"`
+	ModifiedOn    timetypes.RFC3339 `tfsdk:"modified_on" json:"modified_on,computed" format:"date-time"`
+	StartupTimeMs types.Int64       `tfsdk:"startup_time_ms" json:"startup_time_ms,computed"`
 
 	WorkersScriptMetadataModel
 }
@@ -69,14 +69,14 @@ type WorkersScriptMetadataModel struct {
 	CompatibilityFlags *[]types.String                                                   `tfsdk:"compatibility_flags" json:"compatibility_flags,optional"`
 	KeepAssets         types.Bool                                                        `tfsdk:"keep_assets" json:"keep_assets,optional"`
 	KeepBindings       *[]types.String                                                   `tfsdk:"keep_bindings" json:"keep_bindings,optional"`
-	Logpush            types.Bool                                                        `tfsdk:"logpush" json:"logpush,optional"`
+	Logpush            types.Bool                                                        `tfsdk:"logpush" json:"logpush,computed_optional"`
 	MainModule         types.String                                                      `tfsdk:"main_module" json:"main_module,optional"`
 	Migrations         customfield.NestedObject[WorkersScriptMetadataMigrationsModel]    `tfsdk:"migrations" json:"migrations,computed_optional"`
 	Observability      customfield.NestedObject[WorkersScriptMetadataObservabilityModel] `tfsdk:"observability" json:"observability,computed_optional"`
 	Placement          customfield.NestedObject[WorkersScriptMetadataPlacementModel]     `tfsdk:"placement" json:"placement,computed_optional"`
 	// Tags               *[]types.String                                                       `tfsdk:"tags" json:"tags,optional"`
 	TailConsumers customfield.NestedObjectList[WorkersScriptMetadataTailConsumersModel] `tfsdk:"tail_consumers" json:"tail_consumers,computed_optional"`
-	UsageModel    types.String                                                          `tfsdk:"usage_model" json:"usage_model,optional"`
+	UsageModel    types.String                                                          `tfsdk:"usage_model" json:"usage_model,computed_optional"`
 }
 
 type WorkersScriptMetadataAssetsModel struct {

--- a/internal/services/workers_script/schema.go
+++ b/internal/services/workers_script/schema.go
@@ -365,23 +365,10 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				Computed:    true,
 				CustomType:  timetypes.RFC3339Type{},
 			},
-			"placement_mode": schema.StringAttribute{
-				Description: "Enables [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement).",
+			"created_on": schema.StringAttribute{
+				Description: "When the script was created.",
 				Computed:    true,
-				Validators: []validator.String{
-					stringvalidator.OneOfCaseInsensitive("smart"),
-				},
-			},
-			"placement_status": schema.StringAttribute{
-				Description: "Status of [Smart Placement](https://developers.cloudflare.com/workers/configuration/smart-placement).",
-				Computed:    true,
-				Validators: []validator.String{
-					stringvalidator.OneOfCaseInsensitive(
-						"SUCCESS",
-						"UNSUPPORTED_APPLICATION",
-						"INSUFFICIENT_INVOCATIONS",
-					),
-				},
+				CustomType:  timetypes.RFC3339Type{},
 			},
 			"startup_time_ms": schema.Int64Attribute{
 				Computed: true,

